css/shopping-cart.css

```diff
@@ -0,0 +1,118 @@
+
+/* Movil style */
+.shopping-cart {
+  width: var(--width);
+  padding: 0px;
+}
+
+.shopping-cart__top {
+  display: flex;
+  padding-top: 15px;
+  padding-bottom: 24px;
+  gap: 9px;
+  margin: 0 16px; 
+}
+
+.shopping-cart__arrow-right {
+  width: 8px;
+  height: 8px;
+  border-right: 2.3px solid var(--color-soft-gray);
+  border-bottom: 2.3px solid var(--color-soft-gray);
+  transform: rotate(-45deg);
+  margin-top: 5px;
+}
+
+.shopping-cart__text {
+  color: var(--color-soft-gray);
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight);
+  line-height: var(--eighteen-pixel);
+  letter-spacing: -1%;
```

Comment:
Esto se marca como error en el navegador

<img width="319" alt="image" src="https://github.com/user-attachments/assets/c2fe3315-1773-4bb7-b5ab-f4bb16f00455">

```suggestion
  letter-spacing: -0.01em;
```